stdlib/src/python/_cpython.mojo

```diff
@@ -869,12 +1029,14 @@ struct CPython:
         return create_module_fn(module_def_ptr, module_api_version)
 
     # int PyModule_AddFunctions(PyObject *module, PyMethodDef *functions)
```

Comment:
Maybe we can move the comments like this (signature) to above the `get_function` call?

Reply:
Sure, done: https://github.com/msaelices/mojo/commit/6e5b526ed1ae5a21fe996c4f281928ef2ca9ae3e